File: CS/Form1.cs
using AIWizardCustomizationExample.Customization;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraPrinting.Native;
using DevExpress.XtraReports.Wizards;

namespace AIWizardCustomizationExample {
    public partial class Form1 : RibbonForm {
        public Form1() {
            InitializeComponent();
            reportDesigner1.AddService<IWizardCustomizationService>(new WizardCustomizationService());
        }

        protected override void OnLoad(EventArgs e) {
            base.OnLoad(e);
        }

        private void button1_Click(object sender, EventArgs e) {
            reportDesigner1.CreateNewReportWizard();
        }
    }
}

File: VB/Form1.vb
Imports AIWizardCustomizationExample.Customization
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraPrinting.Native
Imports DevExpress.XtraReports.Wizards

Namespace AIWizardCustomizationExample

    Public Partial Class Form1
        Inherits RibbonForm

        Public Sub New()
            InitializeComponent()
            reportDesigner1.AddService(Of IWizardCustomizationService)(New WizardCustomizationService())
        End Sub

        Protected Overrides Sub OnLoad(ByVal e As EventArgs)
            MyBase.OnLoad(e)
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs)
            reportDesigner1.CreateNewReportWizard()
        End Sub
    End Class
End Namespace
